app/src/main/java/net/bonysoft/magicmirror/MainActivity.java

```diff
@@ -43,6 +46,7 @@
     private TextView tweetLabel;
     private TextView weatherTemperatureLabel;
     private TextView todayForecastLabel;
+    private ListView todoList;
```

Comment:
Honestly I was tempted to not even use a `ListView` but just copy the items layout into a dedicated layout. This at least gives us a bit of flexibility, with just the need for a change in the resources file.

Regarding why not a `RecyclerView` the reason is that there is nothing to be recycled here, the number of items would be always contained and using a  `RecyclerView` would definitely be an overkill